woocommerce/wc-admin
# Update usage of lib/number to @woocommerce/number

This builds upon #1399 and the introduction of the new `@woocommerce/number` package.

All usage of `formatNumber` has been updated to use the export from the new package. And also to get around the circular dependency between `currency` and `number`, the use cases of where `formatNumber()` was being called with a type of currency have been updated to use `formatCurrency` directly.

__To Test__
- Click around through the dashboard and reports, and verify all things are still rendering as expected.
